Modules/Sources/Support/UI/Support Conversations/AttachmentListView.swift

```diff
@@ -0,0 +1,234 @@
+import SwiftUI
+import AsyncImageKit
+import PDFKit
+import AVKit
+
+struct SingleImageView: View {
+
+    let url: URL
+
+    @GestureState private var currentZoom = 1.0
+
+    var magnification: some Gesture {
+        MagnifyGesture().updating($currentZoom, body: { newValue, state, transaction in
```

Comment:
It didn't seem to work for me. The views are too small for zooming with gestures. I'd suggest using `.contextMenu` with a details view instead (for long-press) or removing it altogether.